src/es/doramasflix/src/eu/kanade/tachiyomi/animeextension/es/doramasflix/Doramasflix.kt

```diff
@@ -187,26 +196,34 @@ class Doramasflix : ConfigurableAnimeSource, AnimeHttpSource() {
 
     private fun parseEpisodeListJson(jsonLine: String?): List<SEpisode> {
         val jsonData = jsonLine ?: return emptyList()
-        val episodes = mutableListOf<SEpisode>()
-        val data = json.decodeFromString<JsonObject>(jsonData)!!.jsonObject["data"]!!.jsonObject
-        data["listEpisodes"]!!.jsonArray!!.map {
-            val noSeason = it.jsonObject["season_number"]!!.jsonPrimitive!!.content
-            val noEpisode = it.jsonObject["episode_number"]!!.jsonPrimitive!!.content
-            var nameEp = it.jsonObject["name"]!!.jsonPrimitive!!.content
+        val data = json.decodeFromString<JsonObject>(jsonData)?.jsonObject?.get("data")?.jsonObject ?: return emptyList()
```

Comment:
You should use [kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/basic-serialization.md) and serializable classes instead of doing `.jsonObject` everywhere